gem/backends.py

```diff
@@ -41,6 +41,13 @@ def _update_user_from_claims(user, claims):
         user.profile = UserProfile(user=user)
         user.profile.site = Site.objects.get(is_default_site=True)
 
+    # check if the user has an alias
+    # if so. ensure that the alias is eaul to the user_name
+    if user.profile.alias is None or user.profile.alias == "":
+        user.profile.alias = user.first_name
```

Comment:
I think we should probably be using user.username here rather than user.first_name. Otherwise we'd be displaying a users firstname on their comments.